staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1275,7 +1302,17 @@ func TestListContinuation(t *testing.T) {
 	if len(out.Items) != 1 || !reflect.DeepEqual(&out.Items[0], preset[1].storedObj) {
 		t.Fatalf("Unexpected second page: %#v", out.Items)
 	}
+	if transformer.reads != 1 {
+		t.Errorf("expected 4 reads, got %d", transformer.reads)
```

Comment:
I would change all these messages to "unexpected reads: %d", generically, since now they're wrong.

Reply:
Thanks for catching this.  Fixed.